pkg/cosign/verify.go

```diff
@@ -928,6 +946,43 @@ func VerifyBundle(ctx context.Context, sig oci.Signature, rekorClient *client.Re
 	return true, nil
 }
 
+func VerifyTSABundle(ctx context.Context, sig oci.Signature, tsaClient *tsaclient.TimestampAuthority, tsaCertChainPath string) (bool, error) {
+	bundle, err := sig.TSABundle()
+	if err != nil {
+		return false, err
+	} else if bundle == nil {
+		return false, nil
+	}
+
+	b64Sig, err := sig.Base64Signature()
+	if err != nil {
+		return false, fmt.Errorf("reading base64signature: %w", err)
+	}
+
+	fmt.Println("Verifying TSA Bundle")
```

Comment:
Perhaps we want to avoid printing this message (it will be printed per validated signature).

Reply:
If you have it, it should go to STDERR